src/scripts/main.js

```diff
@@ -1,3 +1,57 @@
 'use strict';
 
-// write code here
+const butApRow = document.querySelector('.append-row');
+const butRemRow = document.querySelector('.remove-row');
+const butApCol = document.querySelector('.append-column');
+const butRemCol = document.querySelector('.remove-column');
+const table = document.querySelector('.field');
+
+function update() {
+  const rows = table.rows.length;
+  const cols = table.rows[0].cells.length;
```

Comment:
Ensure that the table has at least one row before accessing `table.rows[0]`. If the table is empty, this line will throw an error. Consider adding a check to ensure the table is not empty before accessing its rows.